### DISCUSSION TITLE
Welcome to "Kotlin MUI Showcase" Discussions!

### DISCUSSION THREAD

**Discussion by aerialist7** [karakum-team/kotlin-mui-showcase#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by viktorhrekh** [comment]
Some showcases look bad with dark mode and white background, is it just my browser problem?

✅

**Reply by aerialist7** [reply]
Yeah. Seems we need to add a background for `Content` component (a MUI `Paper` can be used for this)

**Reply by aerialist7** [reply]
Added, please check

**Reply by viktorhrekh** [reply]
Good work 🎉

**Comment by viktorhrekh** [comment]
@aerialist7 
Check the #6 PR

**Comment by santansarah** [comment]
Great project - thank you! 😀
